week-2/I-array-properties/exercise.js

```diff
@@ -6,7 +6,11 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
+  if (arr.length > 0) {
+    return false; // complete this statement
```

Comment:
This is correct but we can improve the code here to make it a bit shorter. Remember that `arr.length > 0` evaluates to a boolean - that is why you can use it in an `if` statement. But here we're checking that boolean to decide whether to return another boolean. We could just return the original boolean. Could you try fixing that up?